apps/reference/docs/guides/storage.mdx

```diff
@@ -12,7 +12,7 @@ Supabase Storage makes it simple to store and serve large files.
 
 ### Files
 
-Files can be any sort of media file. This includes images, GIFs, and videos. It is best practice to store files outside of your database because of their sizes.
+Files can be any sort of media file. This includes images, GIFs, and videos. It is best practice to store files outside of your database because of their sizes.  HTML files will be returned as plain text for security reasons.
```

Comment:
```suggestion
Files can be any sort of media file. This includes images, GIFs, and videos. It is best practice to store files outside of your database because of their sizes. For security, HTML files are returned as plain text.
```